yarn.lock

```diff
@@ -4971,41 +4971,41 @@ cozy-client-js@^0.15.0:
     pouchdb-browser "7.0.0"
     pouchdb-find "7.0.0"
 
-cozy-client@6.64.5:
-  version "6.64.5"
-  resolved "https://registry.yarnpkg.com/cozy-client/-/cozy-client-6.64.5.tgz#33302c3190387ad42928189ae916f5afb28c3a84"
-  integrity sha512-DnlNaYNTeTAuIAGeKggQbGW/a9CUl19Ec9IdUzBlzkeylAkBvdfbyIg6BTzXePdae0o+bGh0LV9VAJ5yX4C2YA==
+cozy-client@13.5.3, cozy-client@^13.5.3:
+  version "13.5.3"
+  resolved "https://registry.yarnpkg.com/cozy-client/-/cozy-client-13.5.3.tgz#e18dfec09a1c4110036e6399cd14087cd0f25643"
+  integrity sha512-d6CmjHHNDld+oyJJ/dI8Ic3u/IUiKNRHwfFZkPEhgI5k4EeHdzUKfoVOySdsGIylavVtcf6eSMDhgNY0xyVbiA==
   dependencies:
+    btoa "^1.2.1"
     cozy-device-helper "^1.7.3"
-    cozy-stack-client "^6.64.3"
+    cozy-logger "^1.6.0"
+    cozy-stack-client "^13.5.3"
+    isomorphic-fetch "^2.2.1"
     lodash "^4.17.13"
     microee "^0.0.6"
+    minilog "https://github.com/cozy/minilog.git#master"
+    open "^7.0.2"
     prop-types "^15.6.2"
-    react-redux "^5.0.7"
+    react-redux "^7.2.0"
     redux "^3.7.2"
     redux-thunk "^2.3.0"
+    server-destroy "^1.0.1"
     sift "^6.0.0"
     url-search-params-polyfill "^7.0.0"
 
-cozy-client@^13.4.2:
-  version "13.4.2"
-  resolved "https://registry.yarnpkg.com/cozy-client/-/cozy-client-13.4.2.tgz#b495c4da97db2cc97a99fce341ef4d0f037ebe3c"
-  integrity sha512-r2/RGznnov0VmSAk7onnc75UK9cbmNthGwkASOlCnuDZpSYdOxfcCzjIcYQOmEQFuWbOAQzzFMqhju7FYzwtyA==
+cozy-client@6.64.5:
```

Comment:
I'll try to remember to update those :/

```
=> Found "cozy-ach#cozy-client@6.64.5"
info This module exists because "cozy-ach" depends on it.
info Disk size without dependencies: "5.27MB"
info Disk size with unique dependencies: "17.6MB"
info Disk size with transitive dependencies: "21.39MB"
info Number of shared dependencies: 22
=> Found "cozy-jobs-cli#cozy-client@6.58.0"
info Reasons this module exists
   - "cozy-jobs-cli#cozy-konnector-libs" depends on it
   - Hoisted from "cozy-jobs-cli#cozy-konnector-libs#cozy-client"
```